contracts/staking/contracts/src/fees/MixinExchangeFees.sol

```diff
@@ -127,6 +126,20 @@ contract MixinExchangeFees is
         activePoolsThisEpoch[poolId] = pool;
     }
 
+    /// @dev Returns the total balance of this contract, including WETH,
+    ///      minus any WETH that has been reserved for rewards.
+    /// @return totalBalance Total balance.
+    function getAvailableRewardsBalance()
```

Comment:
In the interest of saving codesize, should we remove this function? It wouldn't be difficult to figure this out off-chain if we made `_wethReservedForRewards` public.

Reply:
I like having getters for things that use multiple storage slots, since we can reduce the number of RPC calls we need to make.

Reply:
Won't we have devutils for staking?

Reply:
Alright you got me, removed it in #2190 😎